### ISSUE TITLE
re-write comment/uncomment commands to be a toggleCommand

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#822]
Write one command that checks the start of the text to be commented/uncommented and determine which of the two commands should be called.  This should be easy enough since we can get text styling from scimoz.

This might be a too much work for too little benefit but it might be worth it.
Pros:
- free up a keybinding
  - less keybindings to remember
- consistent with other IDEs
Cons:
- quite a few usecases to consider given you can comment highlighted text, column-select lines, multi line seletion (not column), nothing select (comment/uncomment current line), etc.

**Comment by Naatan** [comment]
I don't think we should be replacing the explicit comment/un-comment commands, as they have their own merit (ie. the fact that they are explicit). But yeah we should definitely have a comment toggle which is bound by default.
